napi/transform/test/transform.test.ts

```diff
@@ -112,6 +112,30 @@ describe('target', () => {
   });
 });
 
+describe('modules', () => {
+  const code = "import * as foo from 'mod';";
+
+  it('should configure esm', () => {
+    const ret = transform('test.js', code, { modules: 'esm' });
+    expect(ret.errors.length).toBe(0);
+    expect(ret.code).toBeDefined();
+    expect(ret.code).toMatchInlineSnapshot(`
+      "import * as foo from "mod";
+      "
+    `);
+  });
+
+  it('should configure cjs', () => {
+    const ret = transform('test.js', code, { modules: 'cjs' });
+    expect(ret.errors.length).toBe(0);
+    expect(ret.code).toBeDefined();
+    expect(ret.code).toMatchInlineSnapshot(`
+      "import * as foo from "mod";
```

Comment:
This is not working as intended because we don't have esm to cjs tranformation yet @Dunqing ?